app/shared/page.tsx

```diff
@@ -0,0 +1,73 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+import styles from "@/styles/FolderPage.module.css";
+import LinkCardComponent from "@/components/LinkCardComponent";
+import { useRequest } from "@/hooks/useRequest";
+import { SampleFolder } from "@/types";
+
+const SharedPage = () => {
+  const [folderInfo, setFolderInfo] = useState<SampleFolder>({});
+
+  const { data, request } = useRequest({
+    url: "api/sample/folder",
+    method: "GET",
+  });
+
+  useEffect(() => {
+    request();
+  }, []);
+
+  useEffect(() => {
+    if (data) {
+      setFolderInfo(data);
+    }
+  }, [data]);
+
+  return (
+    <>
+      <section className={styles.folder_main_section}>
+        <img
+          src={folderInfo?.folder?.owner?.profileImageSource ?? ""}
+          className={styles.folder_user_img}
+        />
+        <p className={styles.folder_user_name}>
+          {folderInfo?.folder?.owner?.name ?? ""}
+        </p>
+        <strong className={styles.folder_name}>
+          {folderInfo?.folder?.name ?? ""}
+        </strong>
+      </section>
+      <section className={styles.folder_info_section}>
+        <div className={styles.folder_search_input_box}>
+          <img src="/images/Search.svg" />
+          <input
+            className={styles.folder_search_input}
+            placeholder="링크를 검색해 보세요"
+          />
+        </div>
+
+        {folderInfo?.folder?.links ? (
+          <article className={styles.folder_card_body}>
+            {folderInfo?.folder?.links?.map((e, i) => {
```

Comment:
50번 라인에서 조건을 검증해줬기 때문에 여기서는 아래와 같이 써줘도 괜찮아요~
```suggestion
            {folderInfo.folder.links.map((e, i) => {
```